### PR TITLE
chore: replaced deprecated method to open TileDB array (#56)

### PR DESCRIPTION
**Pull request by metakuni**
#### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- modify: replaced deprecated `tiledb.Array()` with `tiledb.open()`

### CODE REVIEW

**Review by bkmartinjr — APPROVED** [review]
LGTM!

File: server/dataset/cxg_dataset.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def _validate_and_initialize(self):
 
     @staticmethod
     def _open_array(uri, tiledb_ctx):
-        with tiledb.Array(uri, mode="r", ctx=tiledb_ctx) as array:
+        with tiledb.open(uri, mode="r", ctx=tiledb_ctx) as array:
```
Comment (bkmartinjr) [inline server/dataset/cxg_dataset.py:190]: Reading this more closely, can't this entire function be replaced with a single call to tiledb.open?

Comment (metakuni) [reply server/dataset/cxg_dataset.py:190]: Ha, good point! 😄

**Review by MDunitz — APPROVED** [review]
lgtm

**Pull request merged**